packages/core/datadog-sourcemaps.gradle

```diff
@@ -0,0 +1,139 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
+def reactConfig = project.hasProperty("react") ? project.react : [:]
+def reactRoot = file(reactConfig.root ?: "../../")
+
+def datadogConfig = new Properties()
+def datadogConfigFile = new File(reactRoot, "datadog-sourcemaps.properties")
+
+if (datadogConfigFile.exists()) {
+    datadogConfigFile.withInputStream {
+        datadogConfig.load(it)
+    }
+}
+
+afterEvaluate {
+
+    def androidExtension = extensions.findByName("android")
+
+    if (androidExtension == null) {
+        throw new GradleException("Android extension is missing.")
+    }
+
+    // TODO service name per variant?
+    def serviceName = datadogConfig.SERVICE_NAME ?: androidExtension.defaultConfig.applicationId
+
+    if (serviceName == null) {
+        throw new GradleException("Cannot determine service name.")
+    }
+
+    def releaseVersion = androidExtension.defaultConfig.versionName
+    def buildVersion = androidExtension.defaultConfig.versionCode
+
+    if (releaseVersion == null) {
+        throw new GradleException("Cannot determine application version," +
```

Comment:
Same thing here, I think we should assume that the person reading this message is not familiar with android development.